front/lib/amplitude/node/generated/index.ts

```diff
@@ -94,7 +94,15 @@ export interface AssistantCreatedProperties {
 export interface DataSourceCreatedProperties {
   assistantDefaultSelected: boolean;
   dataSourceName: string;
-  dataSourceProvider: string;
+  dataSourceProvider?: string;
```

Comment:
@PopDaph do you know why `dataSourceProvider` is becoming a optional parameter?

Reply:
The only thing that bothers me by making it optional is that it's easy to forget about it and it's the most important property of the event. But agree that putting a null string isn't ideal neither.

Reply:
This is why I decided to make it optional: 
<img width="597" alt="Screenshot 2024-04-03 at 10 09 40" src="https://github.com/dust-tt/dust/assets/3803406/013dcd5a-6d4a-4cdd-b2cf-03d42a6a3e48">

Reply:
Ok so empty string is seen as "undefined" by Amplitude. thanks.